shared/entity/type.go

```diff
@@ -97,7 +97,8 @@ const (
 	pathPlaceholder = "{pathArgument}"
 )
 
-var entityTypes = []Type{
+// EntityTypes specifies all entity types recognized by LXD.
+var EntityTypes = []Type{
```

Comment:
@markylaing do you see any issues with exporting this? Or is there a better way of iterating the API side entity types?

Reply:
Yes this was intentionally kept private to prevent any external modification. I'd recommend adding a function that returns a copy of the entity types list.

Reply:
ack, thanks